scripts/bench.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/python
+import argparse
+import os
+import sys
+
+from monotonic import monotonic
+
+from detect_secrets.core.usage import PluginOptions
+
+if os.name == 'posix' and sys.version_info[0] < 3:
+    import subprocess32 as subprocess
```

Comment:
I'm actually super surprised by this, because we use `subprocess` quite liberally in the codebase! ([example](https://github.com/Yelp/detect-secrets/blob/caa0e509708c5bfec157f917508d6d0abd25b792/detect_secrets/core/baseline.py#L261))

Can you give an example of when `subprocess32` succeeds when `subprocess` doesn't? Maybe we need to upgrade the other usages as well 🤔

Reply:
Seems like in the second example of #185 `subprocess.call('detect-secrets scan'.split(), stdout=subprocess.DEVNULL)`, on Python2 breaks saying `'module' object has no attribute 'DEVNULL'`! Other GitHub issues recommended using subprocess32 and even the Python 2 wiki recommends using it [[0](https://docs.python.org/2/library/subprocess.html)] as it has better behavior in many situations... But I switched to the other option( using check_output) and now seems to work great with default subprocess. So I removed the subprocess32 dependency as it isn't strictly necessary right now for this case. 

On the other hand, as on the main project, there are many lines using subprocess... maybe further investigation on subprocess vs subprocess32 could offer some improvements to the codebase.

[[0](https://docs.python.org/2/library/subprocess.html)] https://docs.python.org/2/library/subprocess.html